ivadomed/utils.py

```diff
@@ -619,6 +625,52 @@ def threshold_predictions(predictions, thr=0.5):
     return thresholded_preds
 
 
+def segment_volume(model_fname, model_metadata_fname, image_fname, roi_fname=None):
+    """Segment volume.
+    :param model_fname: model filename (.pt) to use.
+    :param model_metadata_fname: model metadata filename (.json), contains model training
+			configuration.
+    :param image_fname: filename of the image to segment.
+    :param roi_fname: filename of a Region Of Interest, used for cropping.
+			e.g. centerline, binary mask.
+    :return: segmented slices.
+    """
+    # Define device
+    device = torch.device("cpu")
+
+    # Load model training config
+    with open(model_metadata_fname, "r") as fhandle:
+        context = json.load(fhandle)
+
+    # Transformations
+    transform_list = []
+    for transform in context['transformation_validation'].keys():
+        parameters = context['transformation_validation'][transform]
+        transform_obj = getattr(ivadomed_transforms, transform)(**parameters)
+        transform_list.append(transform_obj)
+
+    # Load data
+    filename_pairs = [([image_fname], None, roi_fname, None)]
+    if context['unet_3D'] == False:  # TODO: rename this param 'kernel' and create a new param 'model'
```

Comment:
@andreanne-lemay --> what about:
- create a config param `kernel`: either `2d` either `3d`
- create a config param `model`: `unet` or `film` or `hemis`
?

Reply:
Yes, sounds good, but currently film and hemis cannot be used on 3D models. If  we create 2 different params, we should work to make film and hemis 3D compatible ou simply create one param representing the model: unet_3D, unet_2D, film, hemis and attention_unet_3D?